eng/Version.Details.xml

```diff
@@ -14,41 +14,41 @@
     Arcade flow and versions do not necessarily match what is currently consumed by repo build.
   -->
   <ProductDependencies>
-    <Dependency Name="Microsoft.Build" Version="17.3.1">
-      <Uri>https://github.com/dotnet/msbuild</Uri>
-      <Sha>80f618ad45d38475773fd1a6eaa059f118a0ad5a</Sha>
+    <Dependency Name="Microsoft.Build" Version="17.15.0-preview-25257-111">
```

Comment:
we should probably not be updating these versions /cc @ViktorHofer

Reply:
I don't think it matters. Those entries only exist so that the SB infra can flow live / previously-source-built versions in. The version here is ignored.

Reply:
ah I missed that the Version.props entries are unused, I agree then